helpers/search_processing_kendra_quickstart.sh

```diff
@@ -7,7 +7,7 @@ set -o nounset
 
 # Some useful constants
 readonly DOCKER_IMAGE_TAG="opensearch-with-ranking-plugin"
-readonly OPENSEARCH_VERSION="2.5.0"
```

Comment:
Note that we will need to update the quickstart script's template for the plugin zip to reflect the new naming convention. (Of course, until there's a 2.6.0 plugin zip attached to the release, it doesn't matter.)

Reply:
Specifically on line 23, we'll need to make it something like `opensearch-search-processor-2.6.0.zip`, I think.